src/tests/suite/daos_rebuild.c

```diff
@@ -1592,7 +1593,10 @@ rebuild_fail_all_replicas_before_rebuild(void **state)
 	/* Continue rebuild */
 	daos_mgmt_set_params(arg->group, -1, DMG_KEY_FAIL_LOC, 0, 0, NULL);
 
-	sleep(5);
+	/* Sleep long enough to make sure the 2nd rebuild caused by 2nd kill is
```

Comment:
(style)  'nd' may be misspelled - perhaps 'and'?
(style)  'nd' may be misspelled - perhaps 'and'?